enterprise/api_client/enterprise_catalog.py

```diff
@@ -99,6 +100,61 @@ def get_enterprise_catalog(self, catalog_uuid, should_raise_exception=True):
                 )
             return {}
 
+    @JwtLmsApiClient.refresh_token
+    def get_catalog_diff(self, enterprise_customer_catalog, content_keys, should_raise_exception=True):
+        """
+        Gets the representational difference between a list of course keys and the current state of content under an
+        enterprise catalog. This difference is returned as three buckets of data: `items_not_found`,
+        `items_not_included` and `items_found`.
```

Comment:
nit: is `items_found` referring to the `matched_items` used below? Should we converge on one or the other?